common/culture/cultures/wc_goblin.txt

```diff
@@ -8,9 +8,9 @@
 	traditions = {
 		tradition_ancient_miners
 		tradition_maritime_mercantilism
-		tradition_seafaring
```

Comment:
Why you removed `tradition_seafaring`?

Reply:
They have it from their cultural pillar. `heritage_goblin` includes goblins, blackwater and gilgoblin.